source/dialogs/drun.c

```diff
@@ -57,10 +57,20 @@
 
 #define DRUN_CACHE_FILE            "rofi3.druncache"
 #define DRUN_DESKTOP_CACHE_FILE    "rofi-drun-desktop.cache"
+#define XDG_OPEN_COMMAND           "xdg-open"
```

Comment:
Should we make this a config option?

Reply:
I think it makes sense to, absolutely. I didn't include it in this PR because I was looking for a small initial contribution. I'm happy to give it a whirl today. WDYT about `-drun-launcher` as the config option name?

Reply:
Added to the PR a `drun-launcher` config option.